Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -601,10 +601,17 @@ - (void)reactBlur
   [self.backedTextInputView reactBlur];
 }
 
+#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
 - (void)didMoveToWindow
 {
   [self.backedTextInputView reactFocusIfNeeded];
 }
+#else
+- (void)viewDidMoveToWindow
```

Comment:
for this sort of case, can we ifdef just the method name?
#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
- (void)didMoveToWindow
#else
- (void)viewDidMoveToWindow
#endif // TODO(macOS ISS#2323203)

Reply:
That's how we had it initially but @HeyImChris felt it wasn't as readable, so we pivoted to this approach.

Reply:
I guess it's more subjective than I thought but I found this much easier to read.